### PR TITLE
Tasks Documentation 

### PR DESCRIPTION
**Pull request by martinbpeters**

### CODE REVIEW

**Comment by nnevalainen** [comment]
What's the status of this PR? Tasks were introduced in [TSC v0.4](https://github.com/tableau/server-client-python/releases/tag/v0.4) by #159

**Comment by bcantoni** [comment]
@nnevalainen thanks for the reminder ping on this one. The changes look good to me; let me find one more pair of eyes to check.

**Review by bcantoni — APPROVED** [review]

File: docs/api-ref.md [hunk 1]
```diff
@@ -2709,6 +2709,215 @@ The `SubscriptionItem`.  See [SubscriptionItem class](#subscriptionitem-class)
...
+**Attributes**
+
+Name | Description  
+:--- | :---  
+`consecutive_failed_count` | The number of failed consecutive executions.  
+`id` |   The id of the task on the site.  
+`priority` | The priority of the task on the server.
+`schedule_id` | The id of the schedule on the site.    
+`target` | An object, `datasource` or `workbook` which is associated to the task. Source file: models/target.py
+`task_type` | Type of extract task - full or incremental refresh.
```
Comment (bcantoni) [inline docs/api-ref.md:2735]: Suggest reordering these to be in the order shown in TaskItem class above, so id, task_type, ...).

File: docs/api-ref.md [hunk 2]
```diff
@@ -2709,6 +2709,215 @@ The `SubscriptionItem`.  See [SubscriptionItem class](#subscriptionitem-class)
...
+Returns information about the tasks on the specified site.
+
+REST API: [Get Extract Refresh Tasks on Site](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#get_extract_refresh_tasks){:target="_blank"}
+
+
+**Parameters**
+
+Name   |  Description     
+ :--- | : ---    
+`req_option` |  (Optional) You can pass the method a request object that contains additional parameters to filter the request.
```
Comment (bcantoni) [inline docs/api-ref.md:2778]: This should be req_options plural to match

**Review by bcantoni — CHANGES_REQUESTED** [review]
Updating my review with just a couple minor changes needed.

**Comment by nnevalainen** [comment]
Thanks for the prompt reply. Are you @martinbpeters still planning to work on this? If not, how do we want to proceed?  I, for one, can perform the required changes if _allow edits from maintainers_  is checked and I'm granted the necessary permissions.

**Comment by bcantoni** [comment]
@nnevalainen FYI: to wrap this one up, I've merged @martinbpeters original PR and then in a separate commit added my suggested changes from the review. We should be all set here.

**Pull request merged**